models/unit_tests.go

```diff
@@ -32,7 +32,6 @@ func CreateTestEngine(fixturesDir string) error {
 	if err = x.StoreEngine("InnoDB").Sync2(tables...); err != nil {
 		return err
 	}
-	x.ShowSQL(true)
```

Comment:
This change is from your other PR #2995 and unrelated to this PR. It is unlikely to create merge conflicts, but perhaps it should be removed just in case.

Reply:
done